src/utils/card.model.js

```diff
@@ -9,11 +9,11 @@ class Card {
   }
 
   isWizard() {
-    return this.value === 14;
+    return this.value === '14';
```

Comment:
We should be able to leave it as is if we cast it in the game handler ([reference](https://github.com/SE2Project-BHKPTZ/backend/blob/469290c5bf0881165894322bb867f9211ef09317/src/socketHandler/gameHandler.js#L43)).

Reply:
reverted